utils/results/output/securityJobSummary.go

```diff
@@ -9,13 +9,11 @@ import (
 	"github.com/jfrog/gofrog/datastructures"
 	"github.com/jfrog/jfrog-cli-core/v2/commandsummary"
```

Comment:
```suggestion
	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils/commandsummary"
```

The path has changed